docs/components/pages/Calendar.api.md

```diff
@@ -18,11 +18,18 @@ Callback fired when the Calendar navigates between views, or forward and backwar
 
 The minimum date that the Calendar can navigate from.
 
-
 ### max?{ type: 'Date' }
 
 The maximum date that the Calendar can navigate to.
 
+### currentDate?{ type: 'Date', default: 'Date()' }
```

Comment:
this should be: 

`currentDate?{ type: 'Date', default: 'Date()', handler: 'onCurrentDateChange', controllable: true }`